src/Differ.php

```diff
@@ -72,9 +73,9 @@ public function diff($from, $to, LongestCommonSubsequenceCalculator $lcs = null)
      *
      * @param mixed $input
      *
-     * @return string
+     * @return string|array
```

Comment:
why? where do we return array now? and what is inside that array ?

Reply:
this was from a previous commit in another PR which I needed here,
since that PR is merged this is no longer part of this PR.

Reply:
question remains @SpacePossum ...